components/content-service/pkg/layer/provider.go

```diff
@@ -508,13 +529,16 @@ git log --pretty=%H -n 1 > /.workspace/prestophookdata/git_log_2.txt
 cp /workspace/.gitpod/prebuild-log* /.workspace/prestophookdata/
 `
 
+var pvcEnabledFile = `PVC`
+
 // version of this function for persistent volume claim feature
 // we cannot use /workspace folder as when mounting /workspace folder through PVC
 // it will mask anything that was in container layer, hence we are using /.workspace instead here
 func contentDescriptorToLayerPVC(cdesc []byte) (*Layer, error) {
 	layers := []fileInLayer{
 		{&tar.Header{Typeflag: tar.TypeDir, Name: "/.workspace", Uid: initializer.GitpodUID, Gid: initializer.GitpodGID, Mode: 0755}, nil},
 		{&tar.Header{Typeflag: tar.TypeDir, Name: "/.workspace/.gitpod", Uid: initializer.GitpodUID, Gid: initializer.GitpodGID, Mode: 0755}, nil},
+		{&tar.Header{Typeflag: tar.TypeReg, Name: "/.workspace/.gitpod/pvc", Uid: 0, Gid: 0, Mode: 0775, Size: int64(len(pvcEnabledFile))}, []byte(pvcEnabledFile)},
```

Comment:
this allows us to reliably check in supervisor if workspace is using pvc or not, instead of relying on content.json being present, which is not the case everytime (restoring from prebuild will not generate any content.json for example).